test/TestUtilities/Test.Utility/CommandRunner.cs

```diff
@@ -15,133 +15,105 @@ public class CommandRunner
         // Item1 of the returned tuple is the exit code. Item2 is the standard output, and Item3
         // is the error output.
         public static CommandRunnerResult Run(
-            string process,
+            string filename,
             string workingDirectory,
             string arguments,
-            bool waitForExit,
             int timeOutInMilliseconds = 60000,
             Action<StreamWriter> inputAction = null,
             IDictionary<string, string> environmentVariables = null)
         {
-            var psi = new ProcessStartInfo(Path.GetFullPath(process), arguments)
+            var output = new StringBuilder();
+            var error = new StringBuilder();
+
+            using var process = new Process
             {
-                WorkingDirectory = Path.GetFullPath(workingDirectory),
-                UseShellExecute = false,
-                CreateNoWindow = true,
-                RedirectStandardOutput = true,
-                RedirectStandardError = true,
-                RedirectStandardInput = inputAction != null
+                EnableRaisingEvents = true,
```

Comment:
May I know why need to raise event here? 
What is the experience with or without this one? I believe it's improving code quality, I'm curious.